Sources/ClientRuntime/Message/RequestMessageBuilder.swift

```diff
@@ -13,6 +13,10 @@ public protocol RequestMessageBuilder<RequestType>: AnyObject {
 
     init()
 
+    func withHost(_ host: String) -> Self
```

Comment:
`Host` should be provided from `URI`?

Reply:
Oh yea, this is just here so I could use it in a test. I think this can changed to something like `withURI`